adapter/api/proto/wso2/discovery/api/endpoint_security.proto

```diff
@@ -9,11 +9,12 @@ option java_package = "org.wso2.choreo.connect.discovery.api";
 option java_outer_classname = "EndpointSecurityProto";
 option java_multiple_files = true;
 
-// [#protodoc-title: Endpoint]
+// [#protodoc-title: EndpointSecurity]
```

Comment:
we already have a similar message `SecurityInfo` in security_info.proto. can you use it instead of this

Reply:
resolved